tests/test_background.py

```diff
@@ -561,14 +559,10 @@ def testBadPatch(self):
         initialValue = 20
         mi = afwImage.MaskedImageF(500, 200)
         mi.set((initialValue, 0x0, 1.0))
-        im = mi.getImage()
-        im[0:200, :] = np.nan
-        del im
-        msk = mi.getMask()
-        badBits = msk.getPlaneBitMask(
+        mi.image[0:200, :, afwImage.PARENT] = np.nan
```

Comment:
is `afwImage.PARENT` necessary since it's the default? The same goes for the rest of the tests.

Reply:
It's not necessary anymore, but it was when I removed the default in order to rigorously find all the code that relied on it being `LOCAL`.  Now it's just a lot of work to go through and remove it - or rather, I thought it would be back when I thought I was going to put the effort in to determine where I could use `PARENT` instead of `LOCAL` when updating the code.  I didn't keep that up, so it actually looks like it'd be pretty easy to take out the default here, at least.